device-connectors/src/testflinger_device_connectors/devices/zapper/__init__.py

```diff
@@ -99,3 +101,6 @@ def _run(self, zapper_ip, *args, **kwargs):
             logger=logger,
             **kwargs,
         )
+
+    def _post_run_actions(self, args):
+        """Run further actions after Zapper API returns succesfully."""
```

Comment:
Is it supposed to be an empty function?

Reply:
Default implementation is an empty function yes, it's up to the implementor to eventually override it. I'm not marking it as abstract like the other one because it's not mandatory to implement it.